mod_form.php

```diff
@@ -293,6 +293,96 @@ public function definition() {
             $mform->addElement('checkbox', 'completionactivities', get_string('completionissuecheckbox', 'accredible'));
         }
 
+        $attributemappingdefaultvalues =
```

Comment:
This will need to be updated to use templates during multiple attributes implementation